client/packages/host/src/Admin/Help.tsx

```diff
@@ -0,0 +1,23 @@
+import {
+  AppBarButtonsPortal,
+  Box,
+  useInitialisationStatus,
+} from 'packages/common/src';
```

Comment:
This will need to be updated to `from '@openmsupply-client/common'` ... otherwise if/when we add any tests around this component, the tests will fail due to breaking imports 😢